base_config.yml

```diff
@@ -1,8 +1,11 @@
 ---
 docker_hub_namespace: &docker_hub_namespace ekino
+DOCKER_SOCK_PATH: &DOCKER_SOCK_PATH unix://var/run/docker.sock
```

Comment:
you don't need to declare an anchor (`&DOCKER_SOCK_PATH`) if you don't use the reference anywhere else in a yaml file (`*DOCKER_SOCK_PATH`).